### ISSUE TITLE
devCommand.js:71: Cannot read property 'js' of undefined

### ISSUE THREAD

**Issue by mogami74** [kintone/kintone-cli#9]
When I try kintone-cli dev, I get Error.

```shell
C:\Users\username\Documents\project>kintone-cli dev --app-name some-app-name --watch
Building distributed file...
Starting local webserver...
C:\Users\aibri\AppData\Roaming\npm\node_modules\kintone-cli\dist\commands\Dev\devCommand.js:71
            config.uploadConfig.config.js = config.uploadConfig.config.js.map((item) => {
                                                                       ^

TypeError: Cannot read property 'js' of undefined
    at Socket.ws.stderr.on (C:\Users\aibri\AppData\Roaming\npm\node_modules\kintone-cli\dist\commands\Dev\devCommand.js:71:72)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at Socket.Readable.push (_stream_readable.js:208:10)
    at Pipe.onread (net.js:594:20
```

**Comment by miyass** [comment]
@mogami74 
Thank you for informing about this error.
It was fixed by version of [v1.0.3](https://github.com/kintone/kintone-cli/releases/tag/v0.1.3)!

Could you uninstall old version, and install kintone-cli again?
Maybe version of 1.0.3 will be installed!
Thank you!

**Comment by mogami74** [comment]
Thank you for your info.  It sure worked fo me!
